modules/UI/videolayout/SmallVideo.js

```diff
@@ -519,16 +519,13 @@ SmallVideo.prototype.showDominantSpeakerIndicator = function (show) {
         return;
     }
 
-    var indicatorSpanId = "dominantspeakerindicator";
-    var indicatorSpan = this.getIndicatorSpan(indicatorSpanId);
+    var indicatorSpan = this.getIndicatorSpan({
+        id: 'dominantspeakerindicator',
+        content: '<i id="indicatoricon" class="fa fa-bullhorn"></i>',
+        tooltip: 'speaker'
+    });
 
-    indicatorSpan.innerHTML
-        = "<i id='indicatoricon' class='fa fa-bullhorn'></i>";
-    // adds a tooltip
-    UIUtil.setTooltip(indicatorSpan, "speaker", "top");
-    APP.translation.translateElement($(indicatorSpan));
-
-    $(indicatorSpan).css("visibility", show ? "visible" : "hidden");
+    $(indicatorSpan)[show ? "show" : "hide"]();
```

Comment:
Why we are using jquery's show/hide methods instead of setting css properties? I was thinking that setting the css property is more efficient isn't it?